### PR TITLE
add pandonet and testnet

### PR DESCRIPTION
**Pull request by vishalpando**
resolved the formatting and other errors

### CODE REVIEW

File: _data/chains/eip155-3601.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://www.pandoproject.org/",
+  "shortName": "pando-mainnet",
+  "chainId": 3601,
+  "networkId": 3601,
+  "explorers": [
+    {
+      "name": "Pando Mainnet Explorer",
+      "url": "https://explorer.pandoproject.org",
```
Comment (ligi) [inline _data/chains/eip155-3601.json:18]: does not seem to be EIP3091 compatible - e.g. see:

https://explorer.pandoproject.org/block/4457685

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-3601.json (hunk 1 above)
Comment (loganpando) [reply _data/chains/eip155-3601.json:18]: Hello @ligi https://explorer.pandoproject.org/blocks/4457685 is the correct url

Comment (vishalpando) [reply _data/chains/eip155-3601.json:18]: please refer this url = https://explorer.pandoproject.org/blocks/4457685     , the one you sent is not correct

Comment (ligi) [reply _data/chains/eip155-3601.json:18]: that violates EIP3091

Comment (loganpando) [reply _data/chains/eip155-3601.json:18]: that is fixed now
https://explorer.pandoproject.org/block/4459713
please review

Comment (loganpando) [reply _data/chains/eip155-3601.json:18]: EIP3091 rules are followed please review and update @ligi

**Pull request closed without merging**